lib/features/steps/breadcrumb_steps.rb

```diff
@@ -39,6 +39,20 @@
   raise("Breadcrumb with type: #{type} matched") if found
 end
 
+# Test whether the first event entry does not contain a breadcrumb with a specific type and message.
+# Used for confirming filtering of breadcrumbs
+#
+# @step_input type [String] The type of the breadcrumb expected to be absent
+# @step_input message [String] The message of the breadcrumb expected to be absent
+Then('the event does not have a {string} breadcrumb with message {string}') do |type, message|
+  value = Maze::Server.errors.current[:body]['events'].first['breadcrumbs']
+  found = false
+  value.each do |crumb|
+    found = true if crumb['type'] == type && crumb['metaData'] && crumb['metaData']['message'] == message
```

Comment:
Can be written more succinctly by just assigning the result of the boolean expression to §found§:

```suggestion
    found = crumb['type'] == type && crumb['metaData'] && crumb['metaData']['message'] == message
```

Reply:
In fact, you can go further and just say:
```
found = value.any? { |crumb| crumb['type'] == type && crumb['metaData'] && crumb['metaData']['message'] == message }
```